### DISCUSSION TITLE
Request: Add detect black frames in video file

### DISCUSSION THREAD

**Discussion by Masina86** [SubtitleEdit/subtitleedit#5336] [Ideas]
Basically it should detect the black frames mostly used in TV series . It would make sync of a subtitle much more easier because the black frames of a video file are hardly ever the same length and makes the syncing harder.

It would be cool is someone could add/make this feature. It would go well with "scene change" feature. 

I know it is feature of FFMPEG but it would be nice if it can be added as a stand alone in subtitle edit.

**Comment by OmrSi** [comment]
What about the `Seek silence` feature that is in the waveform?
The black frames shouldn't have any audio, so `Seek silence` should do the trick.
It looks for parts where there is no audio in the file.

**Reply by Masina86** [reply]
Hmm maybe it will work gonna test it.

**Reply by Masina86** [reply]
Yes, it works like a charm. Thank for the fast answer i love Subtitle Edit.
